scipy-stubs/stats/_distn_infrastructure.pyi

```diff
@@ -805,7 +805,8 @@ class rv_continuous(_rv_mixin, rv_generic):
         optimizer: Callable[
             [_FloatND, tuple[float, ...], tuple[float, ...], bool],
             tuple[onp.ToFloat, ...],
-        ],
+        ]
+        | None = None,
```

Comment:
I just tried with `None`, and that seems to work (although I'm not sure why). But since it defaults to `optimize.fmin`, I think it'd be better do use `...` as default here:

```suggestion
        | None = ...,
```